topics/releases.md

```diff
@@ -1,81 +1,103 @@
-is not considered to be production ready: it may contain critical bugs,
-not entirely ready features, and may be unstable.
+This branch is the source tree where most of the new features under development.
+`unstable` is not considered production-ready: it may contain critical bugs, incomplete features, and is potentially unstable features.
 
-However, we try hard to make sure that even the unstable branch is
-usable most of the time in a development environment without major
-issues.
+However, we try hard to make sure that even the unstable branch is usable most of the time in a development environment without significant issues.
 
-Forked, Frozen, Release candidate tree
-===
+### Release Candidate
 
-When a new version of Redis starts to be planned, the unstable branch
-(or sometimes the currently stable branch) is forked into a new
-branch that has the name of the target release.
+New minor and major versions of Redis begin as forks of the `unstable` branch.
+The forked branch's name is the target release
 
-For instance, when Redis 2.6 was released as stable, the `unstable` branch
-was forked into the `2.8` branch.
+For example, when Redis 6.0 was released as stable, the `unstable` branch was forked into the `6.2` branch. The new branch is the release candidate (RC) for that version.
 
-This new branch can be at three different levels of stability:
-development, frozen, and release candidate.
+Bug fixes and new features that can be stabilized during the release's time frame are committed to the release candidate branch.
+That said, the `unstable` branch may include work that is not a part of the release candidate.
 
-* Development: new features and bug fixes are committed into the branch, but not everything going into `unstable` is merged here. Only the features that can become stable in a reasonable time frame are merged.
-* Frozen: no new feature is added, unless it is almost guaranteed to have zero stability impacts on the source code, and at the same time for some reason it is a very important feature that must be shipped ASAP. Big code changes are only allowed when they are needed in order to fix bugs.
-* Release Candidate: only fixes are committed against this release.
+The first release candidate, or RC1, is released once it can be used for development purposes and for testing the new version.
+At this stage, most of the new features and changes the new version brings are ready for review, and the release's purpose is collecting the public's feedback.
 
-Stable tree
-===
+Subsequent release candidates are released every three weeks or so, primarily for fixing bugs.
+These may also add new features and introduce changes, but at a decreasing rate towards the final release candidate.
+
+### Stable Tree
+
+Once development has ended and the frequency of critical bug reports for the release candidate wanes, it is ready for the final release.
+
+At this point, the release is marked as stable.
```

Comment:
maybe mention that this means adding a `.0` instead of the `RC` string to the release name

Reply:
Mentioned.